### ISSUE TITLE
Support angular 1.3.x

### ISSUE THREAD

**Issue by pencilcheck** [vasyabigi/angular-slick#52]

**Comment by rodrigoreis22** [comment]
Are there any issues with angular 1.3?

**Comment by pencilcheck** [comment]
https://github.com/vasyabigi/angular-slick/blob/master/bower.json

Your bower.json specified ~1.2.6 for angular

**Comment by marthinus-engelbrecht** [comment]
Yeah, also need angular 1.3.x would be great if you can upgrade or make the bower.json less restrictive

**Comment by pencilcheck** [comment]
Just do angular ~> 1.3.11 would work

**Comment by marthinus-engelbrecht** [comment]
I'm not sure  what you mean. I've managed to force it to use angular 1.3.13 by executing:
 `bower install angular-slick --save --force-latest`
But that's really not ideal, because every time someone new checks out my project and they do a `bower install` they have to add `--force-latest` which might have other undesired consequences.
